{"name": "test_receive_failure_notification", "status": "failed", "statusDetails": {"message": "AssertionError: No se recibió notificación de fallo\nassert False\n +  where False = <bound method DeliveryPage.check_notification_exists of <tests.pages.delivery_page.DeliveryPage object at 0x000001E80475B110>>()\n +    where <bound method DeliveryPage.check_notification_exists of <tests.pages.delivery_page.DeliveryPage object at 0x000001E80475B110>> = <tests.pages.delivery_page.DeliveryPage object at 0x000001E80475B110>.check_notification_exists", "trace": "tests\\test_notifications.py:30: in test_receive_failure_notification\n    assert delivery_page.check_notification_exists(), \"No se recibió notificación de fallo\"\nE   AssertionError: No se recibió notificación de fallo\nE   assert False\nE    +  where False = <bound method DeliveryPage.check_notification_exists of <tests.pages.delivery_page.DeliveryPage object at 0x000001E80475B110>>()\nE    +    where <bound method DeliveryPage.check_notification_exists of <tests.pages.delivery_page.DeliveryPage object at 0x000001E80475B110>> = <tests.pages.delivery_page.DeliveryPage object at 0x000001E80475B110>.check_notification_exists"}, "description": "\n        P2 #1: Recibir notificación cuando falla la entrega\n        ", "start": 1760550941548, "stop": 1760550953819, "uuid": "c7853f46-8634-496f-aea7-3f159381fa43", "historyId": "680553b39e3df9464fa6b21a7c09dc19", "testCaseId": "680553b39e3df9464fa6b21a7c09dc19", "fullName": "tests.test_notifications.TestNotifications#test_receive_failure_notification", "labels": [{"name": "tag", "value": "test_id('P2-001')"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_notifications"}, {"name": "subSuite", "value": "TestNotifications"}, {"name": "host", "value": "L43429"}, {"name": "thread", "value": "17800-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_notifications"}]}